src/utils/iota.js

```diff
@@ -48,6 +48,9 @@ const selectPollingIndexes = (addresses, numPollingAddresses, bundleSize) => {
  */
 
 const skinnyQueryTransactions = (iotaProvider, addresses) =>
+  iotaProvider &&
+  iotaProvider.api &&
+  iotaProvider.api.findTransactions &&
```

Comment:
i think if api exists, findtransactions should exist?

Reply:
Better to be paranoid lol